test/integration/controller/jobs/tfjob/tfjob_controller_test.go

```diff
@@ -520,6 +522,193 @@ var _ = ginkgo.Describe("Job controller when waitForPodsReady enabled", ginkgo.O
 			},
 		}),
 	)
+
+	ginkgo.When("the queue has admission checks", func() {
```

Comment:
nit: I think we should move this case to pytorchjob_controller_test because we have all the test cases for kubeflowjobs in the integration tests for the PyTorchJob. So other kubeflowjobs including TFJob, don't have all cases.

Reply:
We used to verify only behavior depending on TFJob in the TFJob integration tests.

Reply:
Got it, will move it

Reply:
Moved in the last comment, PTAL

Reply:
It looks good :) Thank you!